test/dsl/dsl_basics.jl

```diff
@@ -63,63 +67,58 @@ end
 
 ### Test Interpolation Within the DSL ###
 
-@parameters α k k1 k2
-@species A(t) B(t) C(t) D(t)
-
+# Tests basic interpolation cases.
 let
-    AA = A
-    AAA = A^2 + B
-    rn = @reaction_network rn begin
-        @parameters k
-        @species A(t) B(t) C(t) D(t)
-        k*$AAA, C --> D
-    end
-    rn2 = ReactionSystem([Reaction(k*AAA, [C], [D])], t; name=:rn)
-    @test rn == rn2
+    # Declares parameters and species used across the test.
+    @parameters α k k1 k2
```

Comment:
Same as stuff on the lhs, Git seems to mark some stuff as new when it shouldn't.